compiler/resolution/lowerIterators.cpp

```diff
@@ -1589,7 +1655,7 @@ static bool expandIteratorInline(ForLoop* forLoop)
     // Replace yield statements in the inlined iterator body with copies
     // of the body of the For Loop that invoked the iterator, substituting
     // the yielded index for the iterator formal.
-    expandBodyForIteratorInline(forLoop, ibody, index);
+    expandBodyForIteratorInline(forLoop, ibody, index, map);
```

Comment:
Not that I am against it, but this `processShadowVariables`, and passing the `map` to this function are similar to what we do with `forall`s today, correct?

Reply:
Correct